Lombiq.Gulp.Extensions.csproj

```diff
@@ -26,7 +26,20 @@
     - Binding InitCommonNodeModules to run before both of those ensures that the NPM-related commands will run towards the
       beginning of the build only once, so there are no concurrent processes locking each other's files.
   -->
-  <Target Name="InitCommonNodeModules" BeforeTargets="ResolveReferences" Inputs="package.json" Outputs="$(InitCommonNodeModulesStampFile)">
+  <Target Name="InitCommonNodeModules"
+          BeforeTargets="ResolveReferences"
+          Inputs="package.json"
+          Outputs="$(InitCommonNodeModulesStampFile)">
+    <!-- Ignore sporadic errors when this is run by multiple threads at the same time. -->
+    <Exec Command="corepack enable"
+          IgnoreExitCode="true"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="$(NxVerbosity)" />
+    <!-- Enable the latest version of pnpm@7. -->
+    <Exec Command="corepack prepare pnpm@latest-7 --activate"
+          IgnoreExitCode="true"
+          IgnoreStandardErrorWarningFormat="true"
+          StandardOutputImportance="$(NxVerbosity)" />
```

Comment:
I've contemplated this addition quite a bit now, and I've come to the conclusion that this would be best handled in `MasterCare.PowerBI` itself, because it's really out of scope of this project.

So, create a Target in `MasterCare.PowerBI.csproj` that will have a `BeforeTargets=InitCommonNodeModules` attribute, and do the `corepack` calls there. You can combine them into one, too:

```
corepack enable && corepack prepare pnpm@latest-7 --activate
```

Furthermore, I think we should replace the use of GE with NE in a next step, but we'll leave that for a separate issue.

Reply:
As I checked the `msbuild` action, it builds GE before any other projects. I think moving the `pnpm` version selection away from here will result in an error.

I'm sure, but I'll give it a try.

Reply:
Ah!

Then let's just use the latest pre-release of NE, please!

Reply:
> But to ease the pain, let's just use the latest pre-release of NE from NuGet.

Do you mean together with GE now?

Reply:
I meant instead of, but if you get it to work with GE, then let's finish this issue like that.

Reply:
Finally, I added the `pnpm` version selection target to GE.
Without it, the build result in an error. See: https://github.com/Lombiq/MasterCare/actions/runs/4564731521/jobs/8054878060#step:5:451

Migrating to NE is a bigger impact I guess, so I won't do it under this issue.

Reply:
I'm sorry for the inconvenience.

But it looks like the version cap will benefit a few other projects of ours that are still stuck with GE.